### PR TITLE
API calls

### LINKED ISSUE #4: UI: API calls

### PR DESCRIPTION
**Pull request by Kristoffermg**
Endpoints included in this version:
- Insert
- Range query
- KNN

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -16,6 +19,15 @@ def endpoint():
     else:
         return "Unknown item"
 
+# @app.route('/insert', methods=['POST'])
+# def insert_trajectory():
+#     data = request.get_json()
+    
+#     x = requests.post("http://localhost:8080/insert", json = data)
+
+#     return jsonify(data), 200
```
Comment (KarmaKamikaze) [inline app.py:28]: Det er WIP, eller skal det slettes?

File: templates/index.html [hunk 2]
```diff
@@ -57,33 +66,125 @@ <h1>Trace-Q trajectory simplifier</h1>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
     <script>
```
Comment (KarmaKamikaze) [inline templates/index.html:67]: Kan man ikke definere javascript delen i scripts i filer, så det er lettere at finde rundt i?

File: .idea/.gitignore
Comment (Flaiizor) [inline .idea/.gitignore:1]: Det er lidt pinligt at se 🤓

**Review by KarmaKamikaze — COMMENTED** [review]
Looks gooood! Det kunne være fedt, hvis vi skiftede til nodejs, når vi alligevel bruger javascript. Derudover så tilføj `.idea` mappen til .gitignore og slet den fra commits.

File: app.py (hunk 1 above)
Comment (Kristoffermg) [reply app.py:28]: Det skal bare slettes når der alligevel skiftes til NodeJS senere

File: templates/index.html (hunk 2 above)
Comment (Kristoffermg) [reply templates/index.html:67]: Yep yep

File: app.py (hunk 1 above)
Comment (Flaiizor) [reply app.py:28]: Jeg glæder mig til at det bliver lavet om 🤓

File: templates/index.html (hunk 2 above)
Comment (Flaiizor) [reply templates/index.html:67]: Godt brølet! 🥇

**Review by Flaiizor — APPROVED** [review]
Greetings Team,

I am thrilled to announce my wholehearted approval for the proposed GitHub merge, infused with Rizz, Ohio, and even a touch of Skibidi Toilet flair! 🌟

The code enhancements reflect a harmonious blend of innovation and precision, akin to the artistry of Rizz and the charm of Ohio. Let's embrace these additions with the playful spirit of Skibidi Toilet dance moves! 💃🚽

With a resounding "sigma" of agreement, I eagerly anticipate the integration of these exciting changes into our project.

Keep up the outstanding work and creativity!

Best regards,

[Your Name]

File: .idea/.gitignore
Comment (Kristoffermg) [reply .idea/.gitignore:1]: Best regards,

[Your Name]

**Pull request merged**